Gen3_Source_injection.ipynb

```diff
@@ -0,0 +1,602 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    repo = '/global/cfs/cdirs/lsst/production/gen3/DC2/Run3.1i/repo'
</pre><p>can you also describe in words where and from where this pointer goes to (i.e. in NERSC towards DC2 products etc. These paths might change in the future and knowing to what type of products these point can be useful</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/DC2-notebooks/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>